docs/stemcells.md

```diff
@@ -0,0 +1,25 @@
+# Docker Stemcells
+
+Fissile uses docker images based on [BOSH stemcells](http://bosh.cloudfoundry.org/docs/stemcell.html) when compiling packages
+and as a base for all the role images that it creates.
+
+Prior to version `5.0.0`, fissile would build its own compilation and stemcell
+layers, using hardcoded scripts, that only worked for Ubuntu Trusty.
+
+Starting with version `5.0.0`, fissile no longer builds these images. The user
+is responsible with providing an image that contains the same packages and
+dependencies as a regular [BOSH stemcell](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/tree/master/stemcell_builder/stages), plus:
+- prerequisites: `libopenssl-devel`, `gettext-tools`
+- [`configgin`](https://rubygems.org/gems/configgin) - a ruby gem that processes
+  BOSH erb templates (this means ruby and bundler need to be there as well)
+- [`dumb-init`](https://github.com/Yelp/dumb-init) - a tool used as PID 1 for managing processes in a container
+
+## Implementations
+
+- [OpenSUSE](https://github.com/SUSE/fissile-stemcell-openSUSE/blob/42.2/Dockerfile)
+- [Ubuntu](https://github.com/cloudfoundry-community/fissile-stemcell-ubuntu)
+
+Both of these are built from from the actual BOSH stemcells, you can find the
```

Comment:
`from from` -> `from`

> from the actual BOSH stemcells

To be precise it is built from one of the intermediate blobs that are generated when building the actual stemcells, not the stemcell itself. It is said in the next sentence, but at this point it's not considered a stemcell yet, at least in the upstream terminology.